### ISSUE TITLE
Installation of NiChart_Workflows (NCW)

### ISSUE THREAD

**Issue by gurayerus** [CBICA/NiChart_Workflows#3]
Current input to NCW is a csv file with DLMUSE values (+ the csv file with demographic info). This is the use case when users have already run DLMUSE on their data and prepared the DLMUSE csv file.
Alternatively, users can use Nifti images as input. In this case, the first step of the workflow will be to run NiChart_DLMUSE on the images, to create the csv file with DLMUSE values.
The second option is implemented in the workflow and there is a test dataset to run it (vTest2 - it can be run by changing the name of the config file to vTest2 in Snakefile).
However this would require users to install NiChart_DLMUSE package. At installation this package creates inconsistencies with other packages, so I could run it only in two steps.
In summary: the main task is to create an installation script that can run all workflow steps
Note: NiChart_DLMUSE requires GPU usage, so this may also make the installation more challenging

**Comment by spirosmaggioros** [comment]
This is closed!
